### ISSUE TITLE
--window-x from right

### ISSUE THREAD

**Issue by lollita** [Genymobile/scrcpy#2570]
With
scrcpy --window-x 100
i place the window to 100 from left
do it is possible to place the window to 100 from right.
f.e.:
scrcpy --window-x -100

**Comment by rom1v** [comment]
No, it's not possible (-100 request to place at -100 relative to the primary display).
